src/infoview.ts

```diff
@@ -93,6 +93,13 @@ export class InfoProvider implements Disposable {
             }),
             commands.registerTextEditorCommand('lean.infoView.copyToComment',
                 (editor) => this.copyToComment(editor)),
+            commands.registerTextEditorCommand('lean.infoView.toggleUpdating', (editor) => {
+                if (this.stopped) {
+                    this.setMode(this.displayMode);
+                } else {
+                    this.stopUpdating();
+                }
+            })
```

Comment:
Oops, I completely missed this.  You're missing a trailing comma here.  Please install the tslint extension for vscode for the future!